### DISCUSSION TITLE
Login fails on the browser with mobile app installed

### DISCUSSION THREAD

**Discussion by crozet-magenta** [community/community#148186] [Mobile]
Hi,
I'm trying to login to my GH account on the web browser since some features are not fully available on the app, my account has 2fa enabled and I'm logged properly on the app (I have an Android phone)
On the browser when I enter my username and password, there is a system prompt asking me where I want to continue and at the same time a notification from the mobile app to verify the login request. On the system prompt I ask to continue on the browser and not the gh app, then I validate the sign in request from the app which is successfully but on the browser I'm still on the login page in a loading state and if I try to refresh the page I'm still not connected.
What am I doing wrong here ?

![Screenshot_2024-12-30-16-46-36-839_android.jpg](https://github.com/user-attachments/assets/47c5ee07-5928-47fe-b848-a032bf2f06c0)

**Comment by Dnnzz56** [comment]
Yes
